### PR TITLE
Update `to_prod` & `to_sand` lists for Fri aug 11th

### PR DESCRIPTION
**Pull request by eliheuer**
This PR updates the `to_production` and `to_sandbox` lists for Friday August 11th.

### CODE REVIEW

File: to_production.txt [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+ofl/notosansindicsiyaqnumbers # https://github.com/google/fonts/pull/5285
+ofl/notosansinscriptionalpahlavi # https://github.com/google/fonts/pull/5286
+ofl/notosanskr # https://github.com/google/fonts/pull/5532
+ofl/notosansmayannumerals # https://github.com/google/fonts/pull/5288
+ofl/notosanssc # https://github.com/google/fonts/pull/5533
+ofl/notosanstakri # https://github.com/google/fonts/pull/6409
+ofl/notosanstc # https://github.com/google/fonts/pull/5534
+ofl/oswald # https://github.com/google/fonts/pull/6514
 ofl/robotoflex # https://github.com/google/fonts/pull/6484
+ofl/teko # https://github.com/google/fonts/pull/6560
```
Comment (chrissimpkins) [inline to_production.txt:17]: @m4rc1e heading to prod on next data push

File: to_production.txt [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
 
 # Upgrade
 ofl/bricolagegrotesque # https://github.com/google/fonts/pull/6523
 ofl/notosans # https://github.com/google/fonts/pull/5789
-ofl/notosansimperialaramaic # https://github.com/google/fonts/pull/5284
-ofl/notosansmandaic # https://github.com/google/fonts/pull/5282
+ofl/notosanshk # https://github.com/google/fonts/pull/5531
+ofl/notosansindicsiyaqnumbers # https://github.com/google/fonts/pull/5285
+ofl/notosansinscriptionalpahlavi # https://github.com/google/fonts/pull/5286
+ofl/notosanskr # https://github.com/google/fonts/pull/5532
```
Comment (chrissimpkins) [inline to_production.txt:10]: @simoncozens We're going to push the Noto *Sans* HK, KR, TC, SC updates to production

Comment (chrissimpkins) [reply to_production.txt:10]: We'll do another push for the GB18030-2022 updates.  I'm looking into how we will ship these families in Workspace editors.

**Comment by eliheuer** [comment]
I updated this PR here: https://github.com/google/fonts/pull/6614

It adds the following to `to_production`:
```
# Metadata / Description / License
ofl/gulzar # https://github.com/google/fonts/pull/6562
ofl/notoserifkhitansmallscript # https://github.com/google/fonts/pull/6586
```

**Comment by eliheuer** [comment]
merged from this PR https://github.com/google/fonts/pull/6614

**Pull request closed without merging**